composer.json

```diff
@@ -21,7 +21,7 @@
     "spryker/zed-request": "^3.0.0"
   },
   "require-dev": {
-    "phpstan/phpstan": "*",
+    "codeception/module-asserts": "^1.3",
```

Comment:
not used?

Reply:
Used in CI check:
>             - name: Codecept build
              run: vendor/bin/codecept build --ansi

Reply:
you don't run tests, so you don't need build them. So not needed :)